toolkit/scripts/imggen.mk

```diff
@@ -5,7 +5,13 @@ $(call create_folder,$(IMAGEGEN_DIR))
 
 # Resources
 config_name              = $(notdir $(CONFIG_FILE:%.json=%))
-config_other_files       = $(if $(CONFIG_FILE),$(call shell_real_build_only, find $(CONFIG_BASE_DIR)))
+CONFIG_BASE_DIR          = $(dir $(CONFIG_FILE:%.json=%))
+config_other_files      := $(shell grep -E '.json|.sh' $(CONFIG_FILE)| sed 's/\"Path\"\://' | tr ", \n" " " | tr "\"" " " | xargs)
+define fix_filenames
+$(call shell, if [ "$1" = "/" ]; then echo $2; else echo $(CONFIG_BASE_DIR)$2 | tr -d '[:blank:]'; fi)
+endef
+config_other_files_fixed := $(foreach filename,$(config_other_files),$(call fix_filenames,$(shell echo $(filename) | head -c 1), $(filename)))
```

Comment:
Iff this moves to a script, ``$(call shell_real_build_only, ...)`` might be a good fit. That means we don't execute it for dry-runs (like tab-completing the makefile options). If you keep the current `define` function approach I its probably simpler to just leave it as a normal shell